include/picongpu/fields/incidentField/Solver.hpp

```diff
@@ -125,7 +126,7 @@ namespace picongpu
                     auto const exPosition = traits::FieldPosition<cellType::Yee, FieldE>{}()[0];
                     PMACC_VERIFY_MSG(
                         exPosition[0] == 0.5_X,

```

Comment:
Offline discussed: We should check all components because PIConGPU is allowed to define arbitrary grid positions for the E field and B field even if it mostly makes no sense.
@sbastrakov suggested checking that the grid type is yee grid is much better.

Reply:
I tried now and recalled why i didn't go for checking the grid type. When initially developing incident field solver, at this place there was a problem with inclusion order / cyclic dependencies and I couldn't make it compile. I believe it's possible to work around it by adding more indirection layers, but i thought it's not worth it.

Reply:
ok so we will stay with this test, it's ok!